README.md

```diff
@@ -2,7 +2,8 @@
 
 [MBK keycap profile](https://fkcaps.com/keycaps/mbk) has been designed by **mburger** and is being manufactured by [FKcaps](https://fkcaps.com). They are low profile keycaps, meant to fit Kailh Choc switches.
 
-> These models are provided for ***rendering purpose only*** and are not licensed for any commercial use.
+> These models are provided for ***rendering purpose only*** and are not licensed for any other commercial use than renderings.
+> If you require other formats you may convert them to suite your needs in other software like mesh mixer but these convert files may only only be for personal use and may not be shared. 
```

Comment:
good point, one small thing to fix please:

"only only"

Reply:
fixed